catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/common/TreUtility.java

```diff
@@ -62,7 +64,7 @@ public static String convertToString(Tre tre, String fieldName) {
     @Nullable
     public static Integer convertToInteger(Tre tre, String fieldName) {
         String value = TreUtility.getTreValue(tre, fieldName);
-        if (value != null) {
+        if (value != null && StringUtils.isNotEmpty(value)) {
```

Comment:
I think that StringUtils.isNotEmpty already checks for nonnull https://commons.apache.org/proper/commons-lang/javadocs/api-2.6/org/apache/commons/lang/StringUtils.html#isNotEmpty(java.lang.String)